yt/visualization/base_plot_types.py

```diff
@@ -151,11 +150,20 @@ def _create_axes(self, axrect: tuple[float, float, float, float]) -> None:
 
     def _get_canvas_classes(self):
         if self.interactivity:
-            key = str(matplotlib.get_backend()).lower()
+            key = str(matplotlib.get_backend())
```

Comment:
going down the rabbit hole, I was wondering why we would convert to `str`. I found that the only value that *may* be returned by `matplotlib.get_backend` that is not a string is `None`, and since we don't have a `'none'` key in `BACKEND_SPECS`, I don't see the point of casting.
```suggestion
            key = matplotlib.get_backend()
```
In fact this possibility was only introduced very recently (matplotlib 3.10).


xref #29039